.github/workflows/ci.yml

```diff
@@ -330,125 +152,42 @@ jobs:
-            build-docs: 'ON'
-            compiler-desc: msvc16.11
             cxx-standard: 17
-            vfx-cy: 2022
-            exclude-tests:
-            osver: 2019
-
-          # C++17, Debug -
-          ## - build: 2
-          ##   build-type: Debug
-          ##   build-shared: 'ON'
-          ##   build-docs: 'OFF'
-          ##   cxx-standard: 17
-          ##   exclude-tests:
+            os: windows-latest
 
-          # C++17, Release Static
-          - build: 3
-            build-type: Release
-            build-shared: 'OFF'
-            build-docs: 'OFF'
-            compiler-desc: msvc16.11
-            cxx-standard: 17
-            vfx-cy: 2022
-            exclude-tests:
-            osver: 2019
-
-          # -------------------------------------------------------------------
-          # VFX CY2022 - C++17 - Windows 2022
-          # -------------------------------------------------------------------
-          # C++17, Release Shared
-          - build: 4
-            build-type: Release
+          # Debug
+          - build: 2
+            build-type: Debug
             build-shared: 'ON'
-            build-docs: 'ON'
-            compiler-desc: msvc17.1
-            cxx-standard: 17
-            vfx-cy: 2022
-            exclude-tests:
-            osver: 2022
-
-          # C++17, Debug -
-          ## - build: 5
-          ##   build-type: Debug
-          ##   build-shared: 'ON'
-          ##   build-docs: 'OFF'
-          ##   cxx-standard: 17
-          ##   exclude-tests:
-
-          # C++17, Release Static
-          - build: 6
-            build-type: Release
-            build-shared: 'OFF'
-            build-docs: 'OFF'
-            compiler-desc: msvc17.1
             cxx-standard: 17
-            vfx-cy: 2022
-            exclude-tests:
-            osver: 2022
-
-          # -------------------------------------------------------------------
-          # VFX CY2020 - C++14 - Windows 2019
-          # -------------------------------------------------------------------
-          # C++14, Release Shared
-          - build: 7
-            build-type: Release
-            build-shared: 'ON'
-            build-docs: 'OFF'
-            compiler-desc: msvc16.11
-            cxx-standard: 14
-            vfx-cy: 2020
-            exclude-tests:
-            osver: 2019
-
-          # -------------------------------------------------------------------
-          # VFX CY2019 - C++11 - Windows 2019
-          # -------------------------------------------------------------------
-          # C++11, Release Shared
-          - build: 8
-            build-type: Release
-            build-shared: 'ON'
-            build-docs: 'OFF'
-            compiler-desc: msvc16.11
-            cxx-standard: 11
-            exclude-tests:
-            osver: 2019
+            os: windows-latest
 
+            
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
```

Comment:
It's considered best practices for actions to specify a full hash, not a version tag that could be hijacked. See the OIIO build_steps.yaml for examples.

Reply:
makes sense, thank you